### PR TITLE
Correct ztunnel-config command

### PR DESCRIPTION
**Pull request by xtayfjpk**
## Description

Correct "ztunnel-config" command by removing sub command "experimental"

## Reviewers

- [X] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

https://istio.io/latest/docs/ambient/usage/troubleshoot-waypoint/

### CODE REVIEW

**Comment by xtayfjpk** [comment]
@MorrisLaw Could you review this PR?

**Review by hanxiaop — APPROVED** [review]

File: content/en/docs/ambient/usage/troubleshoot-waypoint/index.md
Comment (ilrudie) [inline content/en/docs/ambient/usage/troubleshoot-waypoint/index.md:38]: It looks like there are some other instances where we still have experimental for ztunnel-config. Would you be able to take a wider pass on finding these?

```suggestion
    If your source calls the destination using the service's hostname or IP, use the `istioctl ztunnel-config service` command to confirm your waypoint is used by the destination service. Following the example earlier, the `reviews` service should use the `reviews-svc-waypoint` while all other services in the `default` namespace should use the namespace `waypoint`.
```

**Review by ilrudie — COMMENTED** [review]
Thank you for the PR. It's a good catch.

File: content/en/docs/ambient/usage/troubleshoot-waypoint/index.md
Comment (xtayfjpk) [reply content/en/docs/ambient/usage/troubleshoot-waypoint/index.md:38]: I only focus on https://istio.io/latest/docs/ambient/usage/troubleshoot-waypoint/, it seems all instances are corrected in this page.

**Review by MorrisLaw — APPROVED** [review]

**Comment by xtayfjpk** [comment]
Hi, who is istio/wg-docs-maintainers-english? no body review?

File: content/en/docs/ambient/usage/troubleshoot-waypoint/index.md
Comment (dhawton) [reply content/en/docs/ambient/usage/troubleshoot-waypoint/index.md:38]: I would like to see what Ian suggested on a wider pass on finding/resolving these. The PR title suggests this is a case.

**Comment by MorrisLaw** [comment]
> Hi, who is istio/wg-docs-maintainers-english? no body review?

If you navigate to @istio/wg-docs-maintainers-english you'll see the list of reviewers. It seems like @dhawton is already looking at this PR and left a comment. Address his comment and get his approval and you should be good to go.

File: content/en/docs/ambient/usage/troubleshoot-waypoint/index.md
Comment (ilrudie) [reply content/en/docs/ambient/usage/troubleshoot-waypoint/index.md:38]: Just to be clear, I didn't mean the entirety of istio.io needs to be scoured and fixed. I do think at least the page being edited should be corrected entirely though.

**Comment by dhawton** [comment]
> Hi, who is istio/wg-docs-maintainers-english? no body review?

I left a comment, please resolve first.

File: content/en/docs/ambient/usage/troubleshoot-waypoint/index.md
Comment (MorrisLaw) [reply content/en/docs/ambient/usage/troubleshoot-waypoint/index.md:38]: Yeah, seems like the 4 cases of `experimental` have been updated on this page now @dhawton

**Review by craigbox — APPROVED** [review]
LGTM

File: content/en/docs/ambient/usage/troubleshoot-waypoint/index.md
Comment (craigbox) [reply content/en/docs/ambient/usage/troubleshoot-waypoint/index.md:38]: I scoured all of istio.io and this was all I found

**Pull request merged**
